documentation/dotnet-trace-instructions.md

```diff
@@ -0,0 +1,132 @@
+# Trace for performance analysis utility (dotnet-trace)
+
+The dotnet-trace is a cross-platform CLI global tool that enables the collection of .NET Core traces of a running process without any native profiler involved. It is built around the EventPipe technology of the .NET Core runtime as an alternative to ETW on Windows and LTTNG on non-Windows.
+
+## Installing dotnet-trace
+
+The first step is to install the dotnet-trace CLI global tool. This requires the 2.1 .NET Core SDK to be installed.
+
+<!-- TODO: Make package globally available. -->
+```cmd
+$ dotnet tool install --global dotnet-trace --version 1.0.3-preview5.19217.3 --add-source https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
+You can invoke the tool using the following command: dotnet-trace
+Tool 'dotnet-trace' (version '1.0.3-preview5.19217.3') was successfully installed.
+```
+
+## Using dotnet-trace
+
+In order to collect traces you will need to:
+
+- First, find out the process identifier (pid) of the .NET Core 3.0 app to collect traces from
```

Comment:
Is it worth fleshing out the `ports` subcommand and just putting that in the rel bits instead of just debug?  That would be simpler to use than these instructions.

Reply:
I could update the design docs, and add it outside the DEBUG build.

Reply:
I'd suggest getting a doc in place with functionality we already have, then we can iterate on any small improvements.